apps/web-portal/config.mjs

```diff
@@ -13,3 +13,9 @@
  * generate all confme file as a post-install task directly in app directory
  *
  */
+
+export const config = {
```

Comment:
maybe will use .env lib according to the 12 factors principle?

Reply:
Even further, take a look on todos above it's about using confme for that